### PR TITLE
Renamed directory names

### PR DESCRIPTION
**Pull request by melroy89**
Fix folder names.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,13 +8,13 @@ OCA banking payment addons for Odoo
 On version 11.0, this project focus on payment interface. The indentation below 
 indicates the dependency graph of the main modules.
 
-- account_banking_payment_export - Basic export functionality of payment orders
+-  `account_banking_pain_base` - Basic export functionality of payment orders (depends on `account_payment_order` & `account_payment_partner`)
```
Comment (pedrobaeza) [inline README.md]: It should be `account_payment_order`

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (melroy89) [reply README.md]: Uh.. The 'base' functionality of the SEPA modules are 'account_banking_pain_base', right? Not account_payment_order, besides the fact that account_banking_pain_base is also depending on account_payment_order, what I wrote between brackets.

Comment (pedrobaeza) [reply README.md]: But this is not for SEPA, but for making payment orders for banks. American banks don't use SEPA for example.

**Review by pedrobaeza — APPROVED** [review]

**Comment by melroy89** [comment]
Let's also do fast tracking here 👍

**Comment by pedrobaeza** [comment]
OK, I can assume that "risk" :wink:

**Pull request merged**